db/migrate/20171114152513_fix_visible_to_departmental_editors_field_for_taxons.rb

```diff
@@ -0,0 +1,17 @@
+class FixVisibleToDepartmentalEditorsFieldForTaxons < ActiveRecord::Migration[5.1]
+  def up
+    published_taxons = Edition.where(document_type: "taxon").where(state: "published")
+    content_ids = published_taxons.joins(:document).pluck(:content_id)
+
+    published_taxons.each do |t|
+      updated_details = t.details.merge(visible_to_departmental_editors: true)
+      t.update(details: updated_details)
+    end
+
+    if Rails.env.production?
+      Commands::V2::RepresentDownstream.new.call(content_ids)
```

Comment:
You'll need to include the line `disable_ddl_transaction!` at the top of the class to do this.

Reply:
👌 thanks, added it!